sympy/calculus/util.py

```diff
@@ -995,17 +996,17 @@ class AccumulationBounds(AtomicExpr):
     calculations, use ``mpmath.iv`` instead.
     """
 
-    is_real = True
+    is_extended_real = True
 
     def __new__(cls, min, max):
 
         min = _sympify(min)
         max = _sympify(max)
 
         inftys = [S.Infinity, S.NegativeInfinity]
-        # Only allow real intervals (use symbols with 'is_real=True').
-        if not (min.is_real or min in inftys) \
-           or not (max.is_real or max in inftys):
+        # Only allow real intervals (use symbols with 'is_extended_real=True').
+        if not (min.is_extended_real or min in inftys) \
```

Comment:
Is `min in inftys` redundant? `oo.is_extended_real` should be true.

Reply:
Yes it is redundant

Reply:
I've removed the infinity check